pkg/domain/infra/abi/containers.go

```diff
@@ -356,6 +356,12 @@ func (ic *ContainerEngine) ContainerRm(ctx context.Context, namesOrIds []string,
 		return reports, nil
 	}
 
+	if options.Force && options.Timeout != nil {
+		// Ignore errors, ic.removeContainer will catch them
+		parallelctr.ContainerOp(ctx, ctrs, func(c *libpod.Container) error {
+			return c.StopWithTimeout(*options.Timeout)
```

Comment:
Can this be moved inside the other ContainerOp function below. I don't think it is necessary to stop all container before we start to remove them.
Also should we really ignore all errors?

Reply:
Moved